.travis.yml

```diff
@@ -20,6 +20,8 @@ env:
     - KUBE=1.14.10
     - KUBE=1.15.7
     - KUBE=1.16.4
+    - KUBE=1.17.0
+    - KUBE=1.18.0
```

Comment:
I'm concerned this will make our test matrix to large. If travis doesn't schedule enough workers, tests are executed sequentially with a wait time.

Reply:
can we drop 1.14.x ? or we might see how it behaves and come back to it

Reply:
I will block this PR until we create a story for the nightly travis. It is taking too much time already.